src/dashboard/frontend/src/components/GodView/confluence/confluence.css

```
@@ -0,0 +1,264 @@
+}
+
+.confluence-trace {
+  position: relative;
+  flex: 1;
+  overflow: hidden;
+}
+
+.confluence-trace canvas { display: block; width: 100%; height: 100%; }
+.confluence-roles { display: flex; width: 170px; flex: 0 0 auto; flex-direction: column; justify-content: center; gap: 2px; padding: 7px 10px; overflow: auto; }
+.confluence-roles span { display: flex; align-items: center; gap: 6px; color: var(--gv-text-secondary); font: 9px var(--gv-font-mono); }
+.confluence-roles i { width: 8px; height: 8px; border-radius: 50%; background: var(--gv-blue); box-shadow: 0 0 6px currentColor; }
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)